arrays-intermediate.js

```diff
@@ -29,11 +34,11 @@ function findFirstStringStartingWithLetter(letter, strings) {
   
   Hint: Use the .map() and .includes() methods
 **************************************************************/
-function isPresentIncluded(presentName, presents) {
-  //TODO: Add your code here
-}
+// function isPresentIncluded(presentName, presents) {
+// return presents.includes(presentName)
```

Comment:
Here, you're supposed to use the `map` method to make all the names of the presents lower cased, and then use the includes on the lower cased present.